contribute/e2e/CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+## INTRODUCTION TO OPENEBS E2E
+
+OpenEBS e2e is focused on workload simulation & application deployment on OpenEBS storage, predominantly in a 
+kubernetes environment. In its current state, it includes application workflow tests,basic compliance and resiliency tests. 
+The scope of the said tests is expected to evolve over time to include performance and security tests as well. 
+The e2e is mostly written in ansible, i.e., as ansible playbooks with docker used for test images (these include standard 
+application images from dockerhub as well as custom-built images stored in openebs/test-storage).Since the tests are performed 
+in a kubernetes environment, the test images are typically deployed as "pods" and "jobs" and therefore accompanied by the 
+respective specification YAML files.
+
+## BUILDING BLOCKS OF OPENEBS CI
+
+The OpenEBS Continuous Integration has two distinct parts : 
+
+- Travis is used to run unit tests, build code and push images to dockerhub (in case of repos where the image is the build artifact)
+- Jenkins pulls these images, creates the kubernetes-based test environment and executes the e2e. It is also used to run tests upon commits that change application specs OR openebs kubernetes deployment specs.
+
+While the OpenEBS e2e can be run as a standalone suite by users, the priority is have them run upon commits to the core openebs github
+repositories such as openebs/openebs, openebs/mayaserver, openebs/jiva. This part of the continuous integration is built around a dedicated Jenkins server, which upon detecting commits via a polling mechanism, launches VMs using Vagrant and runs the ansible playbooks to setup the environment and execute e2e tests. The openebs/e2e project focuses on this part of the CI.

```

Comment:
Jenkins server. On detecting commits through a polling mechanism, the server launches VMs.
CI = Continuous Integration should be expanded and within brackets (CI) at the first usage of CI.

Reply:
Done